eth/catalyst/api.go

```diff
@@ -333,8 +333,11 @@ func computePayloadId(headBlockHash common.Hash, params *beacon.PayloadAttribute
 }
 
 // invalid returns a response "INVALID" with the latest valid hash set to the current head.
-func (api *ConsensusAPI) invalid(err error) beacon.PayloadStatusV1 {
+func (api *ConsensusAPI) invalid(err error, parent *types.Block) beacon.PayloadStatusV1 {
```

Comment:
Maybe `latestValid` instead of `parent`, and let the docstring say
```
// invalid returns a response "INVALID" with the latest valid hash set to the supplied latestValid hash if provided, otherwise the current header hash. 
```